setup.py

```diff
@@ -74,11 +74,11 @@ def run(self):
     version=VersionInfo.version,
     description='Tensorflow to ONNX converter',
     setup_requires=['pytest-runner'],
-    tests_require=['graphviz', 'requests', 'parameterized', 'pytest', 'pytest-cov', 'pyyaml'],
+    tests_require=['graphviz', 'parameterized', 'pytest', 'pytest-cov', 'pyyaml'],
     cmdclass=cmdclass,
     packages=find_packages(),
     author='onnx@microsoft.com',
     author_email='onnx@microsoft.com',
     url='https://github.com/onnx/tensorflow-onnx',
-    install_requires=['numpy>=1.14.1', 'onnx>=1.4.1', 'six']
+    install_requires=['numpy>=1.14.1', 'onnx>=1.4.1', 'requests', 'six']
```

Comment:
add requests as a tf2onnx package dependency, i am considering to support an url as converter input as well.

Reply:
@guschmue, do you have concern with this change?

Reply:
good idea